simulation_parameters/Constants.cs

```diff
@@ -467,6 +467,9 @@ public static class Constants
 
     public const string LOGS_FOLDER_NAME = "logs";
 
+    public const float EDITOR_ARROW_OFFSET = 3.5f;
+    public const float EDITOR_ARROW_INTERPOLATE_SPEED = 5.0f;
```

Comment:
These should be placed around line 414 rather than here, because I've tried to keep things grouped a bit.